src/query/functions/src/scalars/geometry.rs

```diff
@@ -67,6 +68,28 @@ pub fn register(registry: &mut FunctionRegistry) {
     ]);
 
     // functions
+    registry.register_passthrough_nullable_1_arg::<GeometryType, GeometryType, _, _>(
```

Comment:
The return type of `st_asgeojson` function is `An OBJECT in GeoJSON format` in Snowflake, we can use `Variant` type in Databend instead. The value of `Geometry` type should always be a binary format.

Reply:
`st_geomfromgeohash` and `st_geompointfromgeohash` function convert Geometry value to JSON format, but other geometry functions convert to ewkb format. This may cause a geometry column has multi types in a table and we need to check the type for every values before read it. So we'd better keep a consistent storage types, and the data display can be converted uniformly base on some settings, like `GEOMETRY_OUTPUT_FORMAT` in snowflake.

Reply:
I need to add GEOMETRY_OUTPUT_FORMAT config in system config,like
```
SET SESSION GEOMETRY_OUTPUT_FORMAT= EWKT;
```

Reply:
Thanks for the changes, can you help change the value of `st_geomfromgeohash` and `st_geompointfromgeohash` function to `ewkb` format by the way? I'm sorry I didn't notice this when I review the previous PR.

Reply:
> Thanks for the changes, can you help change the value of `st_geomfromgeohash` and `st_geompointfromgeohash` function to `ewkb` format by the way? I'm sorry I didn't notice this when I review the previous PR.

No problem,this pr can be suspended for a few days.I need to be familiar with the configuration item code and implement it together with the session config code and then add "SET SESSION GEOMETRY_OUTPUT_FORMAT= EWKT;" with unit test,otherwise the unit test will report error with ewkb.
  
I followed the approach of· get_timezone() to implement the session configuration, but I don't know how to get self.get_settings() in fmt().
```
  pub fn get_timezone(&self) -> Result<String> {
      self.try_get_string("timezone")
  }

impl<'a> Debug for ScalarRef<'a> {
    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
          ScalarRef::Geometry(s) => {
             // how can i get ctx and config settings here?
              match self.get_settings().get_geometry_output_format() {
                   EWKT => bin_to_ewkt,
                   GEO_JSON=>bin_to_json,
                  ......
             }
          }
```

Reply:
Add  [geometry_output_format](https://docs.snowflake.com/en/sql-reference/parameters#geometry-output-format) parameter;